otel-infrastructure-collector/k8s-helm/values.yaml

```diff
@@ -14,10 +15,10 @@ opentelemetry-collector:
   clusterRole:
     name: "otel-infrastructure-collector"
     create: true
-    rules: 
-    - apiGroups: ["", "events.k8s.io"]
-      resources: ["events"]
-      verbs: ["watch", "list"]
+    rules:
+      - apiGroups: ["", "events.k8s.io"]
+        resources: ["events"]
+        verbs: ["watch", "list"]
```

Comment:
These indentation changes across PR are intended?

Reply:
The lint runs to format the yaml file, I can revert if needed 😄

Reply:
All good, I'm just always suspicion of indentation changes in YAMLs 😉, better linted and not linted!